streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/FlinkClusterServiceImpl.java

```diff
@@ -429,13 +430,15 @@ private ShutDownResponse shutdownInternal(FlinkCluster flinkCluster, String clus
   }
 
   private DeployResponse deployInternal(FlinkCluster flinkCluster)
-      throws InterruptedException, ExecutionException, TimeoutException {
+      throws InterruptedException, ExecutionException, TimeoutException, JsonProcessingException {
     DeployRequest deployRequest =
         new DeployRequest(
             flinkEnvService.getById(flinkCluster.getVersionId()).getFlinkVersion(),
             flinkCluster.getFlinkExecutionModeEnum(),
             flinkCluster.getProperties(),
             flinkCluster.getClusterId(),
+            flinkCluster.getFlinkConfig(),
```

Comment:
Please note that FlinkCluster.getFlinkConfig() retrieves the Flink config from a running Flink JobManager instance through the REST API.
NOT the parameters when submitting a flink cluster deployment.